ecommerce/certprep/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -62,22 +62,7 @@
                                     {% trans "Enroll" %}
                                 </a>
                             {% else %}
-                                {% for processor in payment_processors %}
-                                    <button data-track-type="click"
-                                            data-track-event="edx.bi.ecommerce.basket.payment_selected"
-                                            data-track-category="checkout"
-                                            data-processor-name="{{ processor.NAME|lower }}"
-                                            data-track-checkout-type="hosted"
-                                            class="btn payment-button"
-                                            id="{{ processor.NAME|lower }}">
-                                        {% if processor.NAME == 'cybersource' %}
-                                            {% trans "Enroll" %}
-                                        {% elif processor.NAME == 'paypal' %}
-                                            {# Translators: Do NOT translate the name PayPal. #}
-                                            {% trans "Enroll" %}
-                                        {% endif %}
-                                    </button>
-                                {% endfor %}
+                                <p id="support-text">The enrollment link is invalid, please contact Pearson VUE Support.</p>
```

Comment:
This text should be marked for translation, see line 62. 
Why are you using the id "support-text"? @sergivalero20

Reply:
Ok, let me add it. 
Because I thought it would be necessary to change the style. Of course, It depends on the response that I get from Scott.